stdlib/2and3/bdb.pyi

```diff
@@ -0,0 +1,94 @@
+
+from typing import Set, Dict, Iterable, Any, Callable, Tuple, Type, SupportsInt, List, Union, TypeVar, Optional, IO
+from types import FrameType, TracebackType, CodeType
+
+
+_T = TypeVar("_T")
+_TraceDispatch = Callable[[FrameType, str, Any], Any]  # TODO: Recursive type
+_ExcInfo = Tuple[Type[BaseException], BaseException, FrameType]
+
+GENERATOR_AND_COROUTINE_FLAGS: int = ...
+
+class BdbQuit(Exception): ...
+
+class Bdb:
+
+    skip: Set[str]
+    breaks: Dict[str, List[int]]
+    fncache: Dict[str, str]
+    frame_returning: Optional[FrameType]
+    botframe: Optional[FrameType]
+    quitting: bool
+    stopframe: Optional[FrameType]
+    returnframe: Optional[FrameType]
+    stoplineno: int
+
+    def __init__(self, skip: Iterable[str] = ...) -> None: ...
+    def canonic(self, filename: str) -> str: ...
+    def reset(self) -> None: ...
+    def trace_dispatch(self, frame: FrameType, event: str, arg: Any) -> _TraceDispatch: ...
+    def dispatch_line(self, frame: FrameType) -> _TraceDispatch: ...
+    def dispatch_call(self, frame: FrameType, arg: None) -> _TraceDispatch: ...
+    def dispatch_return(self, frame: FrameType, arg: Any) -> _TraceDispatch: ...
+    def dispatch_exception(self, frame: FrameType, arg: _ExcInfo) -> _TraceDispatch: ...
+    def is_skipped_module(self, module_name: str) -> bool: ...
+    def stop_here(self, frame: FrameType) -> bool: ...
+    def break_here(self, frame: FrameType) -> bool: ...
+    def do_clear(self, arg) -> None: ...
+    def break_anywhere(self, frame: FrameType) -> bool: ...
+    def user_call(self, frame: FrameType, argument_list: None) -> None: ...
+    def user_line(self, frame: FrameType) -> None: ...
+    def user_return(self, frame: FrameType, return_value: Any) -> None: ...
+    def user_exception(self, frame: FrameType, exc_info: _ExcInfo) -> None: ...
+    def set_until(self, frame: FrameType, lineno: int = ...) -> None: ...
+    def set_step(self) -> None: ...
+    def set_next(self, frame: FrameType) -> None: ...
+    def set_return(self, frame: FrameType) -> None: ...
+    def set_trace(self, frame: FrameType = ...) -> None: ...
```

Comment:
Remove `= ...`

Reply:
Are you sure? Because the actual signature for both python 2 and 3.6/7 is `def set_trace(self, frame=None)`

Reply:
You're right, not sure why I wrote that.